### ISSUE TITLE
```git pull —depth=``` 2.33.1-r0 completely-catatonic

### ISSUE THREAD

**Issue by 62f** [ish-app/ish#1610]
![image](https://user-images.githubusercontent.com/68332197/139792034-b5272de2-8aca-4ffe-8c8a-58a278c71815.jpeg)
(Build 282)

**Comment by 62f** [comment]
2.26.3-r0 ain't much better!![image](https://user-images.githubusercontent.com/68332197/139798369-8bcdf021-3ed0-45a6-afde-92127b0bc2ff.jpeg)

**Comment by tharant** [comment]
Do `git config --global pack.threads 1` then try again.

**Comment by 62f** [comment]
> Do `git config --global pack.threads 1` then try again.

Yes I've been using that for a long time now, yet somehow depth=# bypasses antihyperthread!
